doc/release-notes.rst

```diff
@@ -12,12 +12,12 @@ versions of the Zephyr project are released approximately every three-months.
 All Zephyr project source code is maintained in a `GitHub repository`_;
 you can either download source as a tar.gz file (see the bottom of the
 `GitHub tagged releases`_ page corresponding to each release), or use
-Git clone and checkout commands, such as:
+Git checkout commands, on an existing Zephyr installation obtained using the
+`west tool <_get_the_code>`.
 
 .. code-block:: shell
-
-   git clone https://github.com/zephyrproject-rtos/zephyr
-   cd zephyr
+   
+   cd zephyrproject/zephyr
```

Comment:
shouldn't the west commands to get the repo be here (replacing the ``git clone`` command)?

Reply:
There's also mention of cloning the Zephyr repo over in https://www.zephyrproject.org/developers/#downloads.  I take it a simple clone is not going to work any more?

Reply:
This text is not in GitHub. It’s separately maintained on the Wordpress site. Once we agree on how to get the source code, I can update that site to reference the instructions.

Reply:
I have now modified this so that it shows clearly that the instructions are valid until 1.13.